recipes/mobie_utils/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "0.2.0" %}
+{% set name = "mobie_utils" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mobie/mobie-utils-python/archive/{{ version }}.tar.gz
+  sha256: 4af9ca43e439824ee15640a1391a662696b83f365a1ec6548d9321e90aff466b
+
+build:
+  number: 0
+  script: "pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  noarch: python
+  entry_points:
+    - mobie.add_image = mobie.image_data:main
+    - mobie.add_registered_source = mobie.registration:main
+    - mobie.add_segmentation = mobie.segmentation:main
+    - mobie.add_traces = mobie.traces:main
+    - mobie.add_open_organelle_dataset = mobie.open_organelle:main
+    - mobie.migrate_dataset = mobie.migration.migrate_dataset:main
+    - mobie.migrate_project = mobie.migration.migrate_project:main
+    - mobie.validate_dataset = mobie.validation.dataset:main
+    - mobie.validate_project = mobie.validation.project:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
```

Comment:
It's a little unclear what the actual runtime dependencies are since they aren't listed in the `setup.py`, but this list doesn't match what's in the upstream repo in the environment file or conda recipe. For example those list `pybdv` but it's not here. I just want to double check that that's ok.

Reply:
All the other dependencies (e.g. `pybdv`) are pulled in through `cluster_tools` (which is a package I am also maintaining); so I only listed the additional dependencies here.
I am not sure if this is best practice, but I personally prefer this because:
- it's a bit tedious to replicate the dependencies here
- pins can get out of sync if they are updated upstream and not here

Happy to change this if there is a better solution.